lib/wallaroo_labs/logging/README.md

```diff
@@ -0,0 +1,92 @@
+
+# Using the Log logging library
+
+## Prerequisites
+
+```
+// For each source file that uses the `Log` primitive
+use "wallaroo_labs/logging"
+
+// If no Pony source files contain a
+//     use "wallaroo_labs/logging"
+// statement, then add the following to access the C FFI
+// functions in the "libwallaroo-logging.a" library.
+use "lib:wallaroo-logging"
+
+```
+
+## Setup
+
+Early in your Wallaroo app, call `Log.set_defaults()` prior to calling any
+of the public API function described below.
+
+## Using the logging C FFI functions
+
+The top of `wallaroo-logging.pony` has several FFI type specs to
+cut-and-paste into your Pony source, as needed
+
+```
+use @l_enabled[Bool](severity: LogSeverity, category: LogCategory)
+use @ll_enabled[Bool](sev_cat: U16)
+use @l[I32](severity: LogSeverity, category: LogCategory, fmt: Pointer[U8] tag, ...)
+use @ll[I32](sev_cat: U16, fmt: Pointer[U8] tag, ...)
+```
+
+The log messages created by the `@l()` and `@ll()` functions are formatted
+using `sprintf(3)`-style formatting conventions.  The one difference
+is that this library will add a trailing newline/'\n' character to the
+format string.
+
+Log a message at `info` severity for the `checkpoint` category in the
+two argument style:
+
+```
+@l(Log.info(), Log.checkpoint(), "Hello, lucky %d".cstring(), 7)
+```
+
+Log a message at `info` severity for the `checkpoint` category in the
```

Comment:
Can you add a note here that you can also pass any severity in directly to `@ll`?